test/suites/wdio.conf.js

```diff
@@ -44,19 +44,19 @@ exports.config = {
 	// Base for browser.url() and Page#openTitle()
 	baseUrl: process.env.MW_SERVER + process.env.MW_SCRIPT_PATH,
 
+	hostname: 'browser',
+	port: 4444,
+	path: '/wd/hub',
 	// ============
 	// Capabilities
 	// ============
+	// https://sites.google.com/a/chromium.org/chromedriver/capabilities
 	capabilities: [ {
-		// https://sites.google.com/a/chromium.org/chromedriver/capabilities
 		browserName: 'chrome',
 		maxInstances: 1,
 		'goog:chromeOptions': {
-			// If DISPLAY is set, assume developer asked non-headless or CI with Xvfb.
-			// Otherwise, use --headless (added in Chrome 59)
-			// https://chromium.googlesource.com/chromium/src/+/59.0.3030.0/headless/README.md
 			args: [
-				...( process.env.DISPLAY ? [] : [ '--headless' ] ),
+				...( process.env.HEADED_TESTS ? [ '--disable-gpu' ] : [ '--headless' ] ),
```

Comment:
Out of curiosity, why is disabling the gpu for headed tests necessary? Is this what's responsible for the slowdown?

Reply:
It's actually not necessary -- thanks for catching that :)